server/mediapublic/models.py

```diff
@@ -451,8 +451,8 @@ def to_dict(self):
             facebook=self.facebook,
             instagram=self.instagram,
             periscope=self.periscope,
-            user_id=self.user_id,
-            organization_id=self.organization_id,
+            user_id=str(self.user_id),
```

Comment:
I'm saying that some data sources may actually set these as strings normally, in which case you don't have to force them to `str`.

Also, strictly you should be using `six.text_type` rather than `str` for compatibility.